infra/testing/activate-sw-safari.js

```diff
@@ -3,15 +3,17 @@ module.exports = async (swUrl) => {
   let error = await global.__workbox.webdriver.executeAsyncScript((swUrl, cb) => {
     const onStateChangePromise = (registration, desiredState) => {
       return new Promise((resolve, reject) => {
-        if (registration.installing === null) {
-          throw new Error('Regstration is not installing');
+        if (desiredState === 'activated' && registration.active) {
```

Comment:
I made some tweaks to this logic because I've seen some flakiness with the latest Safari TP, where `registration.installing` was `null` and this function gave up.

I think it's less prone to race conditions with these changes, since if we have a `registration.active` and the state we care about is `'activated'`, that should be fine.